hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -6687,6 +6687,13 @@ public int getReadLockCount() {
     return lockedRows;
   }
 
+  /**
+   * Stringify a coarse "dump" of {@code RowLock} information.
+   */
+  public void dumpLocks(final StringBuilder sb) {
```

Comment:
Not finished?

Reply:
I ended up moving this implementation to `HRegionServer` instead. Deleted this stub.